diff --git a/ledfx/api/__init__.py b/ledfx/api/__init__.py
index 22d7dbd6..f035e456 100644
--- a/ledfx/api/__init__.py
+++ b/ledfx/api/__init__.py
@@ -12,6 +12,11 @@ _LOGGER = logging.getLogger(__name__)
 
 SNACKBAR_OPTIONS = ["success", "info", "warning", "error"]
 
+@web.middleware
+async def cors_middleware(request, handler):
+    response = await handler(request)
+    response.headers['Access-Control-Allow-Private-Network'] = 'true'
+    return response
 
 @BaseRegistry.no_registration
 class RestEndpoint(BaseRegistry):
@@ -192,6 +197,7 @@ class RestApi(RegistryLoader):
         self._ledfx = ledfx
 
     def register_routes(self, app):
+        app.middlewares.append(cors_middleware)
         methods = ["GET", "PUT", "POST", "DELETE"]
         cors = aiohttp_cors.setup(
             app,
